### PR TITLE
Add support for phpcs

### PR DESCRIPTION
**Pull request by nhoag**
Addresses https://github.com/LionsAd/drupal_ti/issues/12 (plus one typo).

Let me know any changes you'd like to see!

### CODE REVIEW

File: runners/phpcs/script.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# @file
+# phpcs integration - Script step.
+
+set -e $DRUPAL_TI_DEBUG
+
+ARGS=( $DRUPAL_TI_PHPCS_ARGS )
+
+$HOME/.composer/vendor/bin/phpcs "${ARGS[@]}"
```
Comment (WengerK) [inline runners/phpcs/script.sh]: What about also using phpcsp (Code Sniffer Drupal Practice) ?

```
ARGS=( $DRUPAL_TI_PHPCSP_ARGS )
$HOME/.composer/vendor/bin/phpcsp "${ARGS[@]}"
```

Comment (nhoag) [reply runners/phpcs/script.sh]: I'm not familiar with the `phpcsp` binary. Is there something I can read to learn about it?

I made a couple of modifications to provide default arguments and to make it easier to invoke `drupal/coder` standards.

File: runners/phpcs/script.sh [hunk 2]
```diff
@@ -4,6 +4,16 @@
...
 set -e $DRUPAL_TI_DEBUG
 
-ARGS=( $DRUPAL_TI_PHPCS_ARGS )
+if [[ -n $DRUPAL_TI_PHPCS_ARGS ]]; then
+  ARGS=( $DRUPAL_TI_PHPCS_ARGS )
+else
+  ARGS=(
+    "--colors"
+    "--standard=Drupal,DrupalPractice"
+    "--extensions=php,module,inc,install,test,profile,theme,css,info,txt,md"
```
Comment (WengerK) [inline runners/phpcs/script.sh]: I'm not sure you should lint css files :D

**Review by WengerK — CHANGES_REQUESTED** [review]

**Comment by nhoag** [comment]
Closing this as outdated. Thanks for the consideration and feedback for the PR.

**Pull request closed without merging**